packages/sfp-cli/src/core/package/deploymentCustomizers/PicklistEnabler.ts

```diff
@@ -203,13 +209,30 @@ export default class PicklistEnabler implements DeploymentCustomizer {
             FullName: picklistInOrg.FullName,
         };
         SFPLogger.log(`Update picklist for custom field ${picklistToDeploy.FullName}`, LoggerLevel.INFO, logger);
-        try {
-            await conn.tooling.sobject('CustomField').update(picklistToDeploy);
-        } catch (error) {
-            throw new Error(
-                `Unable to update picklist for custom field ${picklistToDeploy.FullName} due to ${error.message}`
-            );
+        let retryCount = 0;
+        const maxRetries = 6;
```

Comment:
Should we do a progressive retry, 
We can use one of our dependency 'bottleneck' (https://www.npmjs.com/package/bottleneck) to activate it